recipes/libsolace/all/test_package/conanfile.py

```diff
@@ -0,0 +1,24 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class SolaceTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        # Current dir is "test_package/build/<build_id>" and CMakeLists.txt is
+        # in "test_package"
+        cmake.configure()
+        cmake.build()
+
+    def imports(self):
```

Comment:
imports should be remove. The `run_environment=True` in `self.run` is able to solve LD_LIBRARY_PATH and all paths for dynamic libs.

Reply:
:man_facepalming: I was sure I did remove it. Apparently, I have not saved the file. Should be fixed now